src/modules/enrollment/components/staffAssignment/UnassignStaffButton.tsx

```diff
@@ -0,0 +1,41 @@
+import LoadingButton from '@mui/lab/LoadingButton';
+import React from 'react';
+import {
+  GetHouseholdStaffAssignmentHistoryDocument,
+  GetHouseholdStaffAssignmentsDocument,
+  useUnassignStaffMutation,
+} from '@/types/gqlTypes';
+
+interface UnassignButtonProps {
+  staffAssignmentId: string;
+}
+
+const UnassignStaffButton: React.FC<UnassignButtonProps> = ({
+  staffAssignmentId,
+}) => {
+  const [unassignStaff, { error, loading }] = useUnassignStaffMutation({
+    variables: {
+      staffAssignmentId: staffAssignmentId,
+    },
+    refetchQueries: [
+      GetHouseholdStaffAssignmentsDocument,
+      GetHouseholdStaffAssignmentHistoryDocument,
+    ],
+    awaitRefetchQueries: true,
+  });
+
+  if (error) throw error;
+
+  return (
```

Comment:
Let's discuss at design session tomorrow? Since this is all already happening within a modal, it might not make sense to use our usual confirmation modal pattern.